src/components/header-card/HeaderCard.jsx

```diff
@@ -1,17 +1,22 @@
 import { useTranslation } from 'react-i18next'
 import { Box, Grid } from '@mui/material'
-import { styles } from '~/components/offer-request-block/OfferRequestBlock.styles'
+import { styles } from '~/components/header-card/HeaderCard.styles'
 import AppButton from '~/components/app-button/AppButton'
 import AppCard from '~/components/app-card/AppCard'
 import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
 import AppDrawer from '../app-drawer/AppDrawer'
 import { useDrawer } from '~/hooks/use-drawer'
-import subject from '~/assets/img/offer-page/subject-icon.svg'
 
-const OfferRequestBlock = ({ userRole }) => {
+const HeaderCard = ({
+  buttonText,
+  description,
+  imageAlt,
+  imageSrc,
+  showButton = true,
```

Comment:
Since we pass `buttonText` only when we want to display the button, we can rely on this variable and omit `showButton`:
```javascript
{buttonText && (
  <AppButton onClick={handleOpenDrawer} sx={styles.button}>
    {t(buttonText)}
  </AppButton>
)}
```